relayer/src/link/pending.rs

```diff
@@ -82,7 +82,7 @@ impl<Chain: ChainHandle> PendingTxs<Chain> {
                 // behavior of handling the OperationalData.
                 trace!(
                     "[{}] putting error response in error event queue: {:?} ",
-                    self,
+                    &relay_path,
```

Comment:
This is better, but I'd rather rework all these `[{}]` constructs into span-wise KVs whenever the opportunity arises, as was partially done in #1491.

Reply:
> rework all these [{}] constructs into span-wise KVs
I'm not entirely clear what this means. What will the resulting log lines look like?

Reply:
> I'm not entirely clear what this means. What will the resulting log lines look like?

That will depend on the tracing backend. One of the benefits of using KVs is that they are converted into similar structural fields in logging backends supporting this (e.g. journald). In the terminal output subscriber that is used by Hermes now, a tracing event in a span created with `span!(tracing::Level::ERROR, "some_op", chain="foo", connection="bar")` will look something like this (with some bold and italic font styling added on an ANSI terminal):

```
some_op{chain=foo connection=bar}: here comes the event message
```

So, the information will still be flattened in this simple backend, but it will be easy to decipher and search for (if a bit more verbose).

Reply:
Let's perhaps do this here:

```rust
trace!(
             chain = %relay_path.chain_id()
            counterparty_chain = %relay_path.counterparty_chain_id,
             port = %relay_path.port_id,
             channel = %relay_path.channel_id,
             "[{}] putting error response in error event queue: {:?} ",
             &relay_path
)
```